tools/update-metadata.lua

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env luajit
+
+-- tool to generate localized metadata
+--
+-- metadata is fetched by F-Droid on each tagged release and used to update
+-- https://f-droid.org/packages/org.koreader.launcher.fdroid/
+--
+-- usage: ./tools/update_metadata.lua
+--
+-- NOTE: title and screenshots are not translated. These resources are located in metadata/en-US
+
+package.path = "frontend/?.lua;" .. package.path
+local _ = require("gettext")
+
+local metadata = {
+    ["short_description.txt"] = _("Ebook reader with support for many formats like PDF, DjVu, EPUB, FB2, CBZ."),
+    ["full_description.txt"] = _([[* portable: runs on embedded devices (Cervantes, Kindle, Kobo, PocketBook), Android and Linux computers. Developers can run a KOReader emulator in Linux and MacOS.
+
+* multi-format documents: supports fixed page formats (PDF, DjVu, CBT, CBZ) and reflowable e-book formats (EPUB, FB2, Mobi, DOC, CHM, TXT). Scanned PDF/DjVu documents can also be reflowed with the built->
```

Comment:
I think something went wrong on these lines, some c/p in a terminal editor that cut them off perhaps?

Reply:
yup, thanks!